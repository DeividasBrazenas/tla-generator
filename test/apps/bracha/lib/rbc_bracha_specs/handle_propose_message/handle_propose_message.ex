defmodule Generated.HandleProposeMessage do
  def handle_message(rbc, {:PROPOSE, from, value} = _msg) do

    broadcaster = rbc.broadcaster
    echo_sent = rbc.echo_sent
    me = rbc.me
    msg_recv = rbc.msg_recv
    output = rbc.output
    peers = rbc.peers
    predicate = rbc.predicate

    ^broadcaster = from

    if (not echo_sent and predicate.(value)) do
      msg_recv = Map.put(msg_recv, {from, :PROPOSE}, true)

      rbc = %{rbc | echo_sent: true, msg_recv: msg_recv}

      msgs = Enum.into(peers, %{}, fn peer_id -> {peer_id, [{:ECHO, me, value}]} end)

      {:ok, rbc, msgs, output}

    else
      msgs = %{}

      {:ok, rbc, msgs, output}

    end
  end
end
