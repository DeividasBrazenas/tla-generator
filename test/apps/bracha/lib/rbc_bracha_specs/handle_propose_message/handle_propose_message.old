-------------------- MODULE handle_propose_message --------------------
EXTENDS Naturals, FiniteSets, Sequences, TLC

CONSTANT CN
CONSTANT FN
CONSTANT Value
CONSTANT NotValue
CONSTANT NULL

(*--algorithm handle_propose_message
variables
  bcNode \in AN,
  from = bcNode,
  value \in (CASE from \in CN -> Value [] from \in FN -> {NotValue}),
  echoSent \in BOOLEAN,
  rbcs = [node_id \in AN |->
       [n |-> N,
       f |-> F,
       me |-> node_id,
       peers |-> AN,
       broadcaster |-> bcNode,
       predicate |-> [v \in AllValues |-> TRUE],
       max_msg_size |-> 1000,
       propose_sent |-> FALSE,
       msg_recv |-> [msg_from_node_id \in AN |-> [msg_type \in MsgTypes |-> FALSE]],
       echo_sent |-> echoSent,
       echo_recv |-> [echo_value \in AllValues |-> [x \in AN |-> FALSE]],
       ready_sent |-> FALSE,
       ready_recv |-> [ready_value \in AllValues |-> [x \in AN |-> FALSE]],
       output |-> NULL]],
  msgs = [node_id \in AN |-> {}],
  _msg = <<"PROPOSE", from, value>>,

define
  AN  == CN \cup FN
  N == Cardinality(AN)
  F == Cardinality(FN)
  MsgTypes == {"PROPOSE", "ECHO", "READY"}
  AllValues == Value \cup {NotValue}
end define;

fair process handle_propose_message \in AN
variables
  rbc = rbcs[self],
  broadcaster = NULL,
  echo_sent = NULL,
  me = NULL,
  msg_recv = NULL,
  output = NULL,
  peers = NULL,
  predicate = NULL,
  result = NULL,
begin
handle_propose_message:
  if _msg[1] /= "PROPOSE" then
    goto Done;
  end if;
  after_condition:

  broadcaster := rbc.broadcaster;
  echo_sent := rbc.echo_sent;
  me := rbc.me;
  msg_recv := rbc.msg_recv;
  output := rbc.output;
  peers := rbc.peers;
  predicate := rbc.predicate;

  if broadcaster /= from then
    goto Done;
  end if;
  after_pin_0:

  if (~echo_sent /\ predicate[value]) then
    if_0:
      msg_recv[from]["PROPOSE"] := TRUE;

      rbc.echo_sent := TRUE
      || rbc.msg_recv := msg_recv;

      msgs := [peer_id \in peers |-> msgs[peer_id] \cup {<<"ECHO", me, value>>}];

      result := <<"ok", rbc, msgs, output>>;

  else
    else_0:

      result := <<"ok", rbc, msgs, output>>;

  end if;

end process;
end algorithm; *)

=======================================================================