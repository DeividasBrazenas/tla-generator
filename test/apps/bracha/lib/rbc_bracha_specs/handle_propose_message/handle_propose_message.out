@!@!@STARTMSG 2262:0 @!@!@
TLC2 Version 2.18 of Day Month 20?? (rev: 6dfec66)
@!@!@ENDMSG 2262 @!@!@
@!@!@STARTMSG 2187:0 @!@!@
Running breadth-first search Model-Checking with fp 104 and seed -2035170570609934082 with 1 worker on 8 cores with 7220MB heap and 64MB offheap memory [pid: 29236] (Windows 11 10.0 amd64, Oracle Corporation 18.0.1.1 x86_64, MSBDiskFPSet, DiskStateQueue).
@!@!@ENDMSG 2187 @!@!@
@!@!@STARTMSG 2220:0 @!@!@
Starting SANY...
@!@!@ENDMSG 2220 @!@!@
Parsing file C:\University\Master's Thesis\tla_generator_v2\test\apps\bracha\lib\rbc_bracha_specs\handle_propose_message\handle_propose_message.tla (file:/c:/University/Master's%20Thesis/tla_generator_v2/test/apps/bracha/lib/rbc_bracha_specs/handle_propose_message/handle_propose_message.tla)
Warning: symbols were renamed.
Parsing file C:\Users\Deividas\AppData\Local\Temp\tlc-6551583628899180610\Naturals.tla (jar:file:/C:/Users/Deividas/.vscode/extensions/alygin.vscode-tlaplus-nightly-2022.12.3006/tools/tla2tools.jar!/tla2sany/StandardModules/Naturals.tla)
Parsing file C:\Users\Deividas\AppData\Local\Temp\tlc-6551583628899180610\FiniteSets.tla (jar:file:/C:/Users/Deividas/.vscode/extensions/alygin.vscode-tlaplus-nightly-2022.12.3006/tools/tla2tools.jar!/tla2sany/StandardModules/FiniteSets.tla)
Parsing file C:\Users\Deividas\AppData\Local\Temp\tlc-6551583628899180610\Sequences.tla (jar:file:/C:/Users/Deividas/.vscode/extensions/alygin.vscode-tlaplus-nightly-2022.12.3006/tools/tla2tools.jar!/tla2sany/StandardModules/Sequences.tla)
Parsing file C:\Users\Deividas\AppData\Local\Temp\tlc-6551583628899180610\TLC.tla (jar:file:/C:/Users/Deividas/.vscode/extensions/alygin.vscode-tlaplus-nightly-2022.12.3006/tools/tla2tools.jar!/tla2sany/StandardModules/TLC.tla)
Semantic processing of module Naturals
Semantic processing of module Sequences
Semantic processing of module FiniteSets
Semantic processing of module TLC
Semantic processing of module handle_propose_message
@!@!@STARTMSG 2219:0 @!@!@
SANY finished.
@!@!@ENDMSG 2219 @!@!@
@!@!@STARTMSG 2185:0 @!@!@
Starting... (2022-12-31 00:04:54)
@!@!@ENDMSG 2185 @!@!@
@!@!@STARTMSG 2189:0 @!@!@
Computing initial states...
@!@!@ENDMSG 2189 @!@!@
@!@!@STARTMSG 2269:0 @!@!@
Computed 2 initial states...
@!@!@ENDMSG 2269 @!@!@
@!@!@STARTMSG 2269:0 @!@!@
Computed 4 initial states...
@!@!@ENDMSG 2269 @!@!@
@!@!@STARTMSG 2269:0 @!@!@
Computed 8 initial states...
@!@!@ENDMSG 2269 @!@!@
@!@!@STARTMSG 2190:0 @!@!@
Finished computing initial states: 14 distinct states generated at 2022-12-31 00:04:55.
@!@!@ENDMSG 2190 @!@!@
@!@!@STARTMSG 2193:0 @!@!@
Model checking completed. No error has been found.
  Estimates of the probability that TLC did not check all reachable states
  because two distinct states had the same fingerprint:
  calculated (optimistic):  val = 1.1E-11
@!@!@ENDMSG 2193 @!@!@
@!@!@STARTMSG 2201:0 @!@!@
The coverage statistics at 2022-12-31 00:04:57
@!@!@ENDMSG 2201 @!@!@
@!@!@STARTMSG 2773:0 @!@!@
<Init line 113, col 1 to line 113, col 4 of module handle_propose_message>: 14:14
@!@!@ENDMSG 2773 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 114, col 12 to line 114, col 24 of module handle_propose_message: 1
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 115, col 12 to line 115, col 24 of module handle_propose_message: 4
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 116, col 23 to line 116, col 76 of module handle_propose_message: 4
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 116, col 28 to line 116, col 38 of module handle_propose_message: 4
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 116, col 43 to line 116, col 47 of module handle_propose_message: 3
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 116, col 52 to line 116, col 62 of module handle_propose_message: 1
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2775:0 @!@!@
  |line 116, col 67 to line 116, col 76 of module handle_propose_message: 1:2
@!@!@ENDMSG 2775 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  ||line 116, col 68 to line 116, col 75 of module handle_propose_message: 1
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 117, col 12 to line 117, col 31 of module handle_propose_message: 7
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2775:0 @!@!@
  line 118, col 21 to line 132, col 35 of module handle_propose_message: 14:56
@!@!@ENDMSG 2775 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 118, col 34 to line 118, col 35 of module handle_propose_message: 14
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2775:0 @!@!@
  line 133, col 19 to line 133, col 41 of module handle_propose_message: 14:56
@!@!@ENDMSG 2775 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 133, col 32 to line 133, col 33 of module handle_propose_message: 14
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 134, col 12 to line 134, col 43 of module handle_propose_message: 14
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2775:0 @!@!@
  line 136, col 18 to line 136, col 45 of module handle_propose_message: 14:56
@!@!@ENDMSG 2775 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 136, col 28 to line 136, col 29 of module handle_propose_message: 14
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2775:0 @!@!@
  line 137, col 26 to line 137, col 47 of module handle_propose_message: 14:56
@!@!@ENDMSG 2775 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 137, col 36 to line 137, col 37 of module handle_propose_message: 14
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2775:0 @!@!@
  line 138, col 24 to line 138, col 45 of module handle_propose_message: 14:56
@!@!@ENDMSG 2775 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 138, col 34 to line 138, col 35 of module handle_propose_message: 14
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2775:0 @!@!@
  line 139, col 17 to line 139, col 38 of module handle_propose_message: 14:56
@!@!@ENDMSG 2775 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 139, col 27 to line 139, col 28 of module handle_propose_message: 14
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2775:0 @!@!@
  line 140, col 23 to line 140, col 44 of module handle_propose_message: 14:56
@!@!@ENDMSG 2775 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 140, col 33 to line 140, col 34 of module handle_propose_message: 14
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2775:0 @!@!@
  line 141, col 21 to line 141, col 42 of module handle_propose_message: 14:56
@!@!@ENDMSG 2775 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 141, col 31 to line 141, col 32 of module handle_propose_message: 14
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2775:0 @!@!@
  line 142, col 20 to line 142, col 41 of module handle_propose_message: 14:56
@!@!@ENDMSG 2775 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 142, col 30 to line 142, col 31 of module handle_propose_message: 14
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2775:0 @!@!@
  line 143, col 24 to line 143, col 45 of module handle_propose_message: 14:56
@!@!@ENDMSG 2775 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 143, col 34 to line 143, col 35 of module handle_propose_message: 14
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2775:0 @!@!@
  line 144, col 21 to line 144, col 42 of module handle_propose_message: 14:56
@!@!@ENDMSG 2775 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 144, col 31 to line 144, col 32 of module handle_propose_message: 14
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 145, col 12 to line 145, col 64 of module handle_propose_message: 14
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2775:0 @!@!@
  |line 145, col 17 to line 145, col 64 of module handle_propose_message: 14:56
@!@!@ENDMSG 2775 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  ||line 145, col 27 to line 145, col 33 of module handle_propose_message: 14
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2772:0 @!@!@
<handle_propose_message_ line 147, col 1 to line 147, col 29 of module handle_propose_message>: 2310:7476
@!@!@ENDMSG 2772 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 147, col 37 to line 147, col 72 of module handle_propose_message: 48048
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 147, col 37 to line 147, col 44 of module handle_propose_message: 40572
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 148, col 40 to line 148, col 58 of module handle_propose_message: 7476
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 149, col 48 to line 149, col 81 of module handle_propose_message: 0
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 150, col 45 to line 150, col 92 of module handle_propose_message: 7476
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 151, col 37 to line 155, col 69 of module handle_propose_message: 7476
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2772:0 @!@!@
<after_condition line 157, col 1 to line 157, col 21 of module handle_propose_message>: 2310:7476
@!@!@ENDMSG 2772 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 157, col 29 to line 157, col 56 of module handle_propose_message: 48048
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 157, col 29 to line 157, col 36 of module handle_propose_message: 40572
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 158, col 29 to line 158, col 95 of module handle_propose_message: 7476
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 159, col 29 to line 159, col 89 of module handle_propose_message: 7476
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 160, col 29 to line 160, col 68 of module handle_propose_message: 7476
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 161, col 29 to line 161, col 86 of module handle_propose_message: 7476
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 162, col 29 to line 162, col 80 of module handle_propose_message: 7476
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 163, col 29 to line 163, col 77 of module handle_propose_message: 7476
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 164, col 29 to line 164, col 89 of module handle_propose_message: 7476
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 165, col 32 to line 165, col 57 of module handle_propose_message: 7476
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 166, col 40 to line 166, col 73 of module handle_propose_message: 0
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 167, col 37 to line 167, col 80 of module handle_propose_message: 7476
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 168, col 29 to line 169, col 66 of module handle_propose_message: 7476
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2772:0 @!@!@
<after_pin_0 line 171, col 1 to line 171, col 17 of module handle_propose_message>: 2310:7476
@!@!@ENDMSG 2772 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 171, col 25 to line 171, col 48 of module handle_propose_message: 48048
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 171, col 25 to line 171, col 32 of module handle_propose_message: 40572
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 172, col 29 to line 172, col 70 of module handle_propose_message: 7476
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 172, col 29 to line 172, col 44 of module handle_propose_message: 7476
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 172, col 49 to line 172, col 70 of module handle_propose_message: 3976
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 173, col 33 to line 173, col 69 of module handle_propose_message: 3976
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 174, col 33 to line 174, col 71 of module handle_propose_message: 3500
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 175, col 25 to line 178, col 46 of module handle_propose_message: 7476
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2772:0 @!@!@
<if_0 line 180, col 1 to line 180, col 10 of module handle_propose_message>: 2107:3976
@!@!@ENDMSG 2772 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 180, col 18 to line 180, col 34 of module handle_propose_message: 44548
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 180, col 18 to line 180, col 25 of module handle_propose_message: 40572
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 181, col 18 to line 181, col 78 of module handle_propose_message: 3976
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 182, col 18 to line 183, col 71 of module handle_propose_message: 3976
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 184, col 18 to line 184, col 103 of module handle_propose_message: 3976
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2775:0 @!@!@
  |line 184, col 26 to line 184, col 103 of module handle_propose_message: 3976:15904
@!@!@ENDMSG 2775 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  ||line 184, col 39 to line 184, col 49 of module handle_propose_message: 3976
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 185, col 18 to line 185, col 94 of module handle_propose_message: 3976
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 186, col 18 to line 186, col 51 of module handle_propose_message: 3976
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 187, col 18 to line 189, col 42 of module handle_propose_message: 3976
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2772:0 @!@!@
<else_0 line 191, col 1 to line 191, col 12 of module handle_propose_message>: 1092:3500
@!@!@ENDMSG 2772 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 191, col 20 to line 191, col 38 of module handle_propose_message: 44072
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 191, col 20 to line 191, col 27 of module handle_propose_message: 40572
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 192, col 20 to line 192, col 94 of module handle_propose_message: 3500
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 193, col 20 to line 193, col 53 of module handle_propose_message: 3500
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 194, col 20 to line 196, col 59 of module handle_propose_message: 3500
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2772:0 @!@!@
<Terminating line 204, col 1 to line 204, col 11 of module handle_propose_message>: 0:175
@!@!@ENDMSG 2772 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 204, col 40 to line 204, col 56 of module handle_propose_message: 14784
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 204, col 40 to line 204, col 47 of module handle_propose_message: 14084
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 204, col 31 to line 204, col 37 of module handle_propose_message: 10143
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 205, col 19 to line 205, col 32 of module handle_propose_message: 175
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2202:0 @!@!@
End of statistics.
@!@!@ENDMSG 2202 @!@!@
@!@!@STARTMSG 2200:0 @!@!@
Progress(17) at 2022-12-31 00:04:57: 30,093 states generated (549,309 s/min), 10,143 distinct states found (185,147 ds/min), 0 states left on queue.
@!@!@ENDMSG 2200 @!@!@
@!@!@STARTMSG 2199:0 @!@!@
30093 states generated, 10143 distinct states found, 0 states left on queue.
@!@!@ENDMSG 2199 @!@!@
@!@!@STARTMSG 2194:0 @!@!@
The depth of the complete state graph search is 17.
@!@!@ENDMSG 2194 @!@!@
@!@!@STARTMSG 2268:0 @!@!@
The average outdegree of the complete state graph is 1 (minimum is 0, the maximum 4 and the 95th percentile is 2).
@!@!@ENDMSG 2268 @!@!@
@!@!@STARTMSG 2186:0 @!@!@
Finished in 3296ms at (2022-12-31 00:04:57)
@!@!@ENDMSG 2186 @!@!@
